src/gatling/simulations/simulations/ET_Simulation.scala

```diff
@@ -66,16 +80,47 @@ class ET_Simulation extends Simulation {
         session
     }
 
+  //defines the Gatling simulation model, based on the inputs
+  def simulationProfile(simulationType: String, numberOfPerformanceTestUsers: Double, numberOfPipelineUsers: Double): Seq[OpenInjectionStep] = {
+    simulationType match {
+      case "perftest" =>
+        if (debugMode == "off") {
+          Seq(
+            rampUsers(numberOfPerformanceTestUsers.toInt) during (testDurationMins minutes)
+          )
+        }
+        else {
+          Seq(atOnceUsers(1))
+        }
+      case "pipeline" =>
+        Seq(rampUsers(numberOfPipelineUsers.toInt) during (2 minutes))
```

Comment:
This should now have a 'dot' in, from 3.9.x => 2.minutes

Reply:
Changed